### ISSUE TITLE
Doesn't handle query parameters in bigquery (@param)

### ISSUE THREAD

**Issue by otrajman** [zeroturnaround/sql-formatter#214]
**Describe the bug**
Formatter throws an error when the query string has a query parameter (valid bigquery string). E.g. 
```
select * from foo where bar = @baz
```
**Expected Output**
```
select * from foo where bar = @baz
```
**Actual Output**
Uncaught TypeError: Cannot read properties of undefined (reading 'value')
    at Tokenizer.tokenize (/home/otrajman/Workspace/fora/node_modules/sql-formatter/src/core/Tokenizer.ts:132:39)
    at StandardSqlFormatter.format (/home/otrajman/Workspace/fora/node_modules/sql-formatter/src/core/Formatter.ts:71:36)
    at format (/home/otrajman/Workspace/fora/node_modules/sql-formatter/src/sqlFormatter.ts:73:33)

**Usage**
node v16.14.0
```
# node node_modules/.bin/ts-node 
> import { format } from 'sql-formatter'
> format('select * from foo where bar = @baz')
```

**Additional context**
Really great to have bigquery support. Temp workaround is to remove '@' from query string before formatting.

**Comment by nene** [comment]
Looks like you're not specifying BigQuery as a dialect. I suggest you try:

```js
import { format } from 'sql-formatter';
format('select * from foo where bar = @baz', { language: "bigquery" });
```

**Comment by nene** [comment]
Though it shouldn't really crash when using the default `language: "sql"`.

**Comment by nene** [comment]
Released 6.0.2 which eliminates the crash.

Though you really should use `language: "bigquery"`, as otherwise the formatting will result in invalid SQL.
